packages/orm/lib/orm.js

```diff
@@ -0,0 +1,235 @@
+const chunkArray = (array, chunkSize) => {
+  if (chunkSize <= 0) chunkSize = 1;
+
+  return array.reduce((accum, item, index) => {
+    const chunkIndex = Math.floor(index / chunkSize);
+
+    if (!accum[chunkIndex]) {
+      accum[chunkIndex] = [];
+    }
+
+    accum[chunkIndex].push(item);
+
+    return accum;
+  }, []);
+};
+
+const runQuery = async ({
```

Comment:
I feel like this should be an internal method (not exported) since it is essentially just a proxy for `executeGraphQL` if someone needs to call this directly they probably don't need to be using this package

Reply:
This is useful if you want to run custom queries or mutations, so for consistency should remain part of the public API.